src/Test.UnitTests.Sarif.WorkItems/SarifWorkItemExtensionsTests.cs

```diff
@@ -17,6 +18,8 @@
 {
     public class SarifWorkItemExtensionsTests
     {
+        private readonly string testSarifFilePath = Path.Combine(Directory.GetCurrentDirectory(), "Test", "Data", "File{0}.sarif");
```

Comment:
>readonly [](http://example.com/codeflow?start=16&length=8)

Please add a comment here explaining that we create the test file path in this way so that we don't use a single, platform-specific file path rendering (i.e., we'll get Windows paths on Windows and Linux running xplat).